RELEASE.md

```diff
@@ -43,7 +43,7 @@ following updates are needed:
 
 - `charts/linkerd2/values.yaml`
 
-   Upgrade the version in `global.proxyInit.image.version`
+   Upgrade the version in `global.proxyInit.image.version` or `global.cniPluginVersion`
```

Comment:
Can you change this file to `charts/linkerd-conttrol-plane/values.yaml` and just instruct to update the `proxyInit.image.version` entry?